rnd/autogpt_server/autogpt_server/blocks/rss-reader-block.py

```diff
@@ -0,0 +1,111 @@
+import time
+from datetime import datetime, timezone
+from typing import Any
+
+import feedparser
+import pydantic
+
+from autogpt_server.data.block import Block, BlockOutput, BlockSchema
+from autogpt_server.data.model import SchemaField
+
+
+class RSSEntry(pydantic.BaseModel):
+    title: str
+    link: str
+    description: str
+    pub_date: datetime
+    author: str
+    categories: list[str]
+
+
+class RSSReaderBlock(Block):
+    class Input(BlockSchema):
+        rss_url: str = SchemaField(
+            description="The URL of the RSS feed to read",
+            placeholder="https://example.com/rss",
+        )
+        start_datetime: datetime = SchemaField(
+            description="The date and time to start looking for posts from on the first loop.",
+            placeholder="2023-06-23T12:00:00Z",
+            default=datetime.now(timezone.utc),
```

Comment:
Default will always use the server time ? should we just set the default to `None` and grab the current time inside the block execution?

Reply:
Thats a good idea, I'll add that now

Reply:
That would break the logic of the system actually. I've set the default as 24hrs ago. So the RSS feed will get all new posts in the last day